<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select_quiz_window</class>
 <widget class="QDialog" name="select_quiz_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Quiz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #f0f0f3;
        border-radius: 20px;
    }
    QFrame#frame {
        background-color: #e0e0e0;
        border-radius: 15px;
        border: 1px solid #b0b0b0;
    }
    QLabel#text_label, QLabel#text_label_2 {
        background-color: #62a0ea;
        border: none;
        border-radius: 10px;
        font-size: 16px;
        font-weight: bold;
        padding: 10px;
        color: white;
    }
    QLabel#text_label_3 {
        background-color: #f5c211;
        border: 1px solid #b0b0b0;
        border-radius: 10px;
        font-size: 16px;
    }
    QComboBox {
        background-color: #33d17a;
        border: none;
        border-radius: 10px;
        padding: 6px;
        font-size: 14px;
    }
    QPushButton#start_quiz_button {
        background-color: #33d17a;
        color: white;
        border: none;
        border-radius: 10px;
        padding: 10px;
        font-size: 14px;
    }
    QPushButton#start_quiz_button:hover {
        background-color: #2dbf6d;
    }
   </string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>520</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose quiz theme to play:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select_quiz_comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="text_label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>         Choose difficulty:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="choose_difficulty_comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="start_quiz_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>480</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Quiz</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_label_3">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>310</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Here you can choose whatever you want!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
